internal/controller/controllers/common.go

```diff
@@ -407,6 +409,31 @@ func isNonePlatformCluster(ctx context.Context, client client.Client, cd *hivev1
 	return isUserManagedNetwork(&clusterInstall), false, nil
 }
 
+// isBaremetalPlatformWithoutMAPI returns true if a cluster is baremetal platform and if in its
+// capabilities it specifies baremetal and doesn't specify MAPI
+func isBaremetalPlatformWithoutMAPI(ctx context.Context, client client.Client, installer bminventory.InstallerInternals, cd *hivev1.ClusterDeployment) (bool, error) {
+	isBaremetalPlatform, err := isBaremetalPlatform(ctx, client, cd)
+	if err != nil {
+		return false, err
+	}
+
+	if !isBaremetalPlatform {
+		return false, nil
+	}
+
+	// Skip adding BMH and Machine to spoke cluster if it's baremetal platform
+	// without MAPI
```

Comment:
This comment doesn't seem relevant here now that the function is being used in multiple places.

Reply:
Yes, good catch! Removed it. Thanks!